### ISSUE TITLE
typeorm-model-shim has no Index decoration export

### ISSUE THREAD

**Issue by monumentyusuf** [typeorm/typeorm#654]
Hello there I'm using typeorm with webpack. In order to fix critical dependency warnings I add `alias` in to my webpack config;
```
alias: {
      typeorm: path.resolve(__dirname, "node_modules/typeorm/typeorm-model-shim")
    }
```

And now I'm getting `typeorm_1.Index is not a function` error and I just realize `typeorm/typeorm-model-shim.js`is not exporting `Index` decoration.

Could you please fix this?
Thanks

**Comment by pleerock** [comment]
That file wasn't updated for a while. I'll make a fix on monday. If you wish you can send a PR - this is an easy fix, you just need to add missing decorators definition [here](https://github.com/typeorm/typeorm/blob/master/extra/typeorm-model-shim.js)

**Comment by pleerock** [comment]
fix will be released in `0.1.0-alpha.33`

**Comment by squalsoft** [comment]
For isomorphic models in nuxt.js i added this lines in config:

`build: {
   ...
    extend(cfg, ctx) {
      
      cfg.resolve.alias.typeorm = path.join(__dirname, '../ts-backend/node_modules/typeorm/typeorm-model-shim.js');
    }
  }`

ts-backend is where typeorm lies.
